database/migrations/2021_02_25_194100_create_bookings_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBookingsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('bookings', function (Blueprint $table) {
+            $table->id();
+            $table->string('slot_number')->nullable();
+            $table->integer('total_participants')->nullable();
+            $table->double('total_amount', 8, 2);
```

Comment:
It must be integer like in https://github.com/tipoff/checkout/blob/main/database/migrations/2020_05_05_050000_create_carts_table.php#L18